src/InstallationValidator.Core/Reporting/FileComparisonResultsTeXBuilder.cs

```diff
@@ -35,7 +35,7 @@ protected FileComparisonResultTeXBuilder(ITeXBuilderRepository builderRepository
       public override void Build(T fileComparisonResult, OSPSuiteTracker buildTracker)
       {
          var subParagraph = new SubParagraph(titleFor(fileComparisonResult));
-         var report = new List<object> {subParagraph, fileComparisonResult.State};
```

Comment:
Instead of reporting the state, I introduce this  object ValidationStateReport that allows me to specify a caption